docs/overview/parallel.rst

```diff
@@ -0,0 +1,28 @@
+********************************
+The sherpa.utils.parallel module
+********************************
+
+.. currentmodule:: sherpa.utils.parallel
+
+.. automodule:: sherpa.utils.parallel
+
+.. versionadded:: 4.16.0
+   Prior to this, these symbols were provided by the
+   :py:mod:`sherpa.utils` module.
+
+   .. rubric:: Constants
+
+   .. autosummary::
+      :toctree: api
+
+      _multi
+      _ncpus
+
```

Comment:
Maybe it's easier to just leave them in the rst like you did here? I also agree that, in this case, we should document it and probably want users to use them. Seems easier to leave as is, then to make a property. It's just unfortunately that they are named that way.

I would suggest to rename them to `_ncpus` -> `cpus` and `_multi` -> `multi`. Since they are put into a new module in this PR, this is our chance to do that. I don't think it's cimplicated enough that it needs to be a proerty, in can just be can attribute.
The existing modules can then `from parallel import ncpu as _ncpu` and we have the same backwards compatibility that we have in the current commit.

Reply:
So I've now renamed these (still accessible as the old forms if importing from the old location of `sherpa.utils`,  but no code in Sherpa itself should be affected by this, but there may be downstream users in CXC).